### PR TITLE
ClusterLoader - Adding timer measurement

### PR DESCRIPTION
**Pull request by krzysied**

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/timer.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+func createTimerMeasurment() measurement.Measurement {
+	return &timer{
+		initialized: false,
+	}
+}
+
+type timer struct {
+	initialized bool
+	startTime   time.Time
+}
```
Comment (shyamjvs) [inline clusterloader2/pkg/measurement/common/timer.go]: Do you really need initialized? Can't you just check if startTime is non-zero for that?

Comment (shyamjvs) [reply clusterloader2/pkg/measurement/common/timer.go]: (assuming time.Time has a default value we can compare for)

File: clusterloader2/pkg/measurement/common/timer.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+// Execute supports two actions. start - which start timer. stop - which stops timer
+// and prints to log time duration between start and stop.
+// Stop action requires label parameter to be provided.
+func (t *timer) Execute(config *measurement.MeasurementConfig) error {
+	action, err := util.GetString(config.Params, "action")
```
Comment (shyamjvs) [inline clusterloader2/pkg/measurement/common/timer.go]: You might want to import util package as _ (to a avoid using `util.`)

File: clusterloader2/pkg/measurement/common/timer.go (hunk 1 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/timer.go]: done.

File: clusterloader2/pkg/measurement/common/timer.go (hunk 2 above)
Comment (krzysied) [reply clusterloader2/pkg/measurement/common/timer.go]: I prefer explicit prefixes. I like to know which functions are external (just by looking).

**Pull request merged**